lib/net/common.js

```diff
@@ -122,12 +122,12 @@ exports.REQUIRED_SERVICES = 0
 exports.USER_AGENT = `/bcash:${pkg.version}/`;
 
 /**
- * Max message size (~4mb with segwit, formerly 2mb)
+ * Max message size (2mb formerly 1mb)
  * @const {Number}
  * @default
  */
 
-exports.MAX_MESSAGE = 1 * 1000 * 1000;
+exports.MAX_MESSAGE = 2 * 1000 * 1000;
```

Comment:
It seems they are not using 1000 based MB, instead 1024.

Reply:
Ah, so it'll be 2097152